frontend/src/Movie/Index/Posters/MovieIndexPoster.tsx

```diff
@@ -214,10 +230,41 @@ function MovieIndexPoster(props: MovieIndexPosterProps) {
         <div className={styles.title}>{qualityProfile.name}</div>
       ) : null}
 
+      {showCinemaRelease && inCinemas ? (
+        <div className={styles.title}>
+          <Icon name={icons.IN_CINEMAS} />{' '}
+          {getRelativeDate(inCinemas, shortDateFormat, showRelativeDates, {
+            timeFormat,
+            timeForToday: false,
+          })}
+        </div>
+      ) : null}
+
+      {showReleaseDate && releaseDateType === 'Released' ? (
+        <div className={styles.title}>
+          <Icon name={icons.DISC} />{' '}
+          {getRelativeDate(releaseDate, shortDateFormat, showRelativeDates, {
+            timeFormat,
+            timeForToday: false,
+          })}
+        </div>
+      ) : null}
+
+      {showReleaseDate && releaseDateType === 'Digital' ? (
+        <div className={styles.title}>
+          <Icon name={icons.MOVIE_FILE} />{' '}
+          {getRelativeDate(releaseDate, shortDateFormat, showRelativeDates, {
+            timeFormat,
+            timeForToday: false,
+          })}
+        </div>
+      ) : null}
+
```

Comment:
Could we simplify this by using a variable for the icon name in so we only need this block once?

Reply:
@mynameisbogdan The problem has not been solved and is still present in the current version. the option is checked and dates are not displayed

![image](https://github.com/Radarr/Radarr/assets/130233722/b34b7633-af93-4f45-9120-88a1b6eb69f4)